ui/src/app/pages/homepage/data-access-requirements.tsx

```diff
@@ -713,9 +684,9 @@ const ControlledTierCard = (props: {profile: Profile, reload: Function, spinnerP
                           enabled={isEligible}
                           text={`${institutionDisplayName} must allow you to access ${ctDisplayName} data`}/>
       {displayEraCommons &&
-         <ControlledTierEraModule profile={profile} eligible={isEligible} spinnerProps={spinnerProps}/>}
-      {unsafeAllowSelfBypass &&
-        <SelfBypassCTComplianceTrainingModule profile={profile} reload={reload} eligible={isEligible} spinnerProps={spinnerProps}/>}
+       <ControlledTierEraModule profile={profile} eligible={isEligible} spinnerProps={spinnerProps}/>}
+      {/* XXX: ctModule should not be hardcoded here; multiple UX's are possible here. */}
+      <ModulesForCard profile={profile} modules={[ctModule]} activeModule={ctModule} spinnerProps={spinnerProps}/>
```

Comment:
@jmthibault79 not sure if there's a planned UX at this point. Here are the possibilities I can see:

1. Add the CT Modules to the "activeModule" selection list. These are optional, so we'd need it to be possible to still **skip** the CT modules if desired; if we used this approach as-is, most likely we'd want to place the ct modules AFTER the DUCC in the active module list. THis is slightly awkward because the user has to complete step (3) before (2). Also, "eligibility" is not a real module. 
2. Allow CT modules to be completed at any time; ignore activeModule. Don't show the "next" UX.
3. Add a separate list of ctModules with their own active selection. Allow the "active" ct module to be completed at any time, but only show the "next" UX after the RT modules are all complete.